jenkins/scripts/integration_delete.sh

```diff
@@ -40,7 +40,14 @@ then
   echo "Executer floating IP ${TEST_EXECUTER_FIP_ID} is deleted."
 
   # Check and delete orphaned floating IPs
-  openstack floating ip list --status "DOWN" --column "ID" -f json | jq --raw-output '.[]."ID"' | xargs -0 openstack floating ip delete || true
+  echo "Checking if there are any existing orphaned floating IPs"
+  ORPHANED_FLOATING_IP_LIST="$(openstack floating ip list --status "DOWN" --column "ID" -f json | jq --raw-output '.[]."ID"')"
+  if [ -z "$ORPHANED_FLOATING_IP_LIST" ]; then
+    echo "Orphaned floating IPs are not found."
+  else
+    echo "Deleting all orphaned floating IPs from ${ORPHANED_FLOATING_IP_LIST}."
```

Comment:
nit, can we remove the variable ORPHANED_FLOATING_IP_LIST from the echo, because it will be a long line when there is a big number of IPs